File: php-breeds/app/models/Fish.php
<?php

Class Fish {
	private $id;
	private $name;
	private $name_es;
	private $specie_id;
	private $points;
	private $total_points;
	private $path_image;
	private $validate;
	private $date;
	private $creator_id;
	
	/**
	 * CONSTRUCTOR
	 */
	function __construct($id){
		$this->id=$id;
	}
	

	/**
	 * SETTERS
	 */
	function set_name($name){
		$this->name=$name;
	}
    function set_name_es($name_es){
		$this->name_es=$name_es;
	}
    function set_specie_id($specie_id){
		$this->specie_id=$specie_id;
	}
	function set_points($points){
		$this->points=$points;
	}
	function set_total_points($total_points){
		$this->total_points=$total_points;
	}
    function set_path_image($path_image){
		$this->path_image=$path_image;
	}
	function set_validate($validate){
		$this->validate=$validate;
	}
    function set_date($date){
		$this->date=$date;
	}
    function set_creator_id($creator_id){
		$this->creator_id=$creator_id;
	}
    /**
	 * GETTERS
	 */
	function get_id(){ 
		return $this->id;
	}
	function get_name(){
		return $this->name;
	}
    function get_name_es(){
		return $this->name_es;
	}
    function get_specie_id(){
		return $this->specie_id;
	}
	function get_points(){
		return $this->points;
	}
	function get_total_points(){
		return $this->total_points;
	}
    function get_path_image(){
		return $this->path_image;
	}
	function get_validate(){
		return $this->validate;
	}
    function get_date(){
		return $this->date;
	}
    function get_creator_id(){
		return $this->creator_id;
	}

    	
	/**
	 * TODTRING
	 */
	function to_String(){
		return"<br><img src=".PATHSERVER.$this->path_image." width=40px>Cat id: ".$this->id.", name: ".$this->name.", path: ".$this->path_image.", creator: ".$this->creator_id;	
	}
}

?>

File: java-ConsumoAPI/src/main/java/com/mycompany/consumoapi/models/Fish.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.consumoapi.models;

/**
 *
 * @author kikem
 */
public class Fish {
    private int id;
    private String name;
    private String name_es;
    private String specie_id;
    private String path_image;
    private String date;
    private int creator_id;

    public Fish(int id, String name, String name_es, String specie_id, String path_image, String date, int creator_id) {
        this.id = id;
        this.name = name;
        this.name_es = name_es;
        this.specie_id = specie_id;
        this.path_image = path_image;
        this.date = date;
        this.creator_id = creator_id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName_es() {
        return name_es;
    }

    public void setName_es(String name_es) {
        this.name_es = name_es;
    }

    public String getSpecie_id() {
        return specie_id;
    }

    public void setSpecie_id(String specie_id) {
        this.specie_id = specie_id;
    }

    public String getPath_image() {
        return path_image;
    }

    public void setPath_image(String path_image) {
        this.path_image = path_image;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(int creator_id) {
        this.creator_id = creator_id;
    }

    @Override
    public String toString() {
        return "Fish{" + "id=" + id + ", name=" + name + ", name_es=" + name_es + ", specie_id=" + specie_id + ", path_image=" + path_image + ", date=" + date + ", creator_id=" + creator_id + '}';
    }

    

       
    
}
